### PR TITLE
🧹 Capitalize comments that feed the docs

### PR DESCRIPTION
**Pull request by tas50**
Generate better docs

Signed-off-by: Tim Smith <tsmith84@gmail.com>

### CODE REVIEW

File: resources/packs/aws/aws.lr.go [hunk 1]
```diff
@@ -24659,6 +24659,7 @@ type AwsEc2Instance interface {
 	Field(string) (interface{}, error)
 	Register(string) error
 	Validate() error
+	Attachments() (NO_TYPE_DETECTED, error)
```
Comment (chris-rock) [inline resources/packs/aws/aws.lr.go]: Your comment changes should not change the generated go code

File: resources/packs/aws/aws.lr [hunk 2]
```diff
@@ -1202,75 +1202,75 @@ private aws.ec2.vgwtelemetry {
...
   volumeType string
-  // time the volume was created
+  // Time the volume was created
   createTime time
 }
 
 // AWS EC2 Instance
-private aws.ec2.instance @defaults("arn state") {
-  // arn for the instance
+private aws.ec2.instance @defaults("arn state") {attachments
```
Comment (chris-rock) [inline resources/packs/aws/aws.lr]: the additional `attachments` is wrong and leads to wrongly-generated code

File: resources/packs/core/core.lr [hunk 3]
```diff
@@ -1,6 +1,6 @@
 option go_package = "go.mondoo.com/cnquery/resources/packs/core"
 
-// Provide contextual info about cnquery and environment
+// Provide contextual info about Mondoo Client/cnquery and environment
```
Comment (chris-rock) [inline resources/packs/core/core.lr]: I would simplify this to:

```
Provide contextual information about MQL runtime and environment
```

**Comment by chris-rock** [comment]
I updated the PR to fix the issues.

**Review by chris-rock — APPROVED** [review]
Thank you @tas50 Great improvements

File: resources/packs/aws/aws.lr [hunk 4]
```diff
@@ -1251,7 +1251,7 @@ private aws.ec2.volume @defaults("arn") {
 }
 
 // AWS EC2 Instance
-private aws.ec2.instance @defaults("arn state") {attachments
+private aws.ec2.instance @defaults("arn state") {
```
Comment (tas50) [inline resources/packs/aws/aws.lr:1254]: Nice catch. No idea how that slipped in.

**Pull request merged**